weblate/auth/migrations/0009_migrate_componentlist.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.7 on 2020-06-11 19:28
+
+from django.db import migrations
+
+
+def migrate_componentlist(apps, schema_editor):
+    Group = apps.get_model("weblate_auth", "Group")
+    db_alias = schema_editor.connection.alias
+    groups = Group.objects.using(db_alias).all()
+    for group in groups:
+        if group.componentlist:
+            group.componentlists.add(group.componentlist)
```

Comment:
Better to do the filtering on database side:

```suggestion
    groups = Group.objects.using(db_alias).filter(componentlist__isnull=False)
    for group in groups:
        group.componentlists.add(group.componentlist)
```